ingest/importer/importer.py

```diff
@@ -25,12 +25,12 @@ def __init__(self, ingest_api):
         self.logger = logging.getLogger(__name__)
 
     def dry_run_import_file(self, file_path, project_uuid=None):
-        spreadsheet_json, template_mgr, errors = self._generate_spreadsheet_json(file_path, project_uuid)
+        spreadsheet_json, template_mgr, errors = self.generate_json(file_path, project_uuid)
         entity_map = self._process_links_from_spreadsheet(template_mgr, spreadsheet_json)
 
         return entity_map
 
-    def _generate_spreadsheet_json(self, file_path, project_uuid=None):
+    def generate_json(self, file_path, project_uuid=None):
```

Comment:
Why has `generate_json` been promoted to public? Do we have tests for this unit?

Reply:
Added.